src/components/footer/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { Link, Image, makeStyles, Text, tokens } from '@fluentui/react-components';
+
+const useClasses = makeStyles({
+  footer: {
+    display: 'flex',
+    justifyContent: 'center',
+    padding: tokens.spacingVerticalXXL,
+    backgroundColor: tokens.colorBackgroundOverlay,
+    width: '100%',
+    boxSizing: 'border-box',
+  },
+  footerContainer: {
+    width: '100%',
+    maxWidth: '1440px',
```

Comment:
```suggestion
    maxWidth: '1440px',
```
// first: define global custom property in the root app component:
export const useGlobalStyles = makeStaticStyles({
  ":root": {
    "--container-max-width": "1440px",
  },
});

// then use it in your components:
const useStyles = makeStyles({
  container: {
    maxWidth: "var(--container-max-width)",
  },
});